uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/directives/leave-request-popup.html

```diff
@@ -204,35 +94,10 @@ <h4 class="modal-title">Leave Request</h4>
               </div>
             </crm-loading>
             <div class="row" ng-show="$ctrl.sickness">
-              <div class="col-xs-12 form-horizontal">
-              <hr>
-                <div class="form-group">
-                  <label class="col-xs-4 control-label">Reason<i class="fa fa-asterisk"></i></label>
-                  <div class="col-xs-8">
-                    <div class="chr_custom-select chr_custom-select--full">
-                      <select name="reason">
-                        <option value="- select -">- select -</option>
-                      </select>
-                    </div>
-                  </div>
-                </div>
-                <div class="form-group">
-                  <div class="col-xs-12">
-                    <div>
-                      <input required type="checkbox" value="certification">
-                      <span>Self certification form required</span>
-                    </div>
-                    <div>
-                      <input required type="checkbox" value="interview">
-                      <span>Back to work interview required</span>
-                    </div>
-                    <div>
-                      <input required type="checkbox" value="work">
-                      <span>Return to work required</span>
-                    </div>
-                  </div>
-                </div>
-              </div>
+              <ng-include src="'/sites/all/modules/civicrm/tools/extensions/civihr/uk.co.compucorp.civicrm.hrleaveandabsences/views/my-leave/request-modal/details/sickness.html'"></ng-include>
```

Comment:
The path should not be used like this, though I am not sure how do it properly. Can you post in L&A room?